src/bsp.mk

```diff
@@ -14,7 +14,7 @@ exec_prefix = $(RTEMS_ROOT)/$(RTEMS_CPU)-rtems$(RTEMS_API)
 include $(RTEMS_ROOT)/make/custom/$(RTEMS_BSP).cfg
 
 DEPFLAGS = -MT $@ -MD -MP -MF $(basename $@).d
-SYSFLAGS = -B $(PROJECT_LIB) -specs bsp_specs -qrtems
```

Comment:
Why is the bsp_specs file gone? Is'nt it needed anymore?

Reply:
The `bsp_specs` have been removed from RTEMS. They are not necessary any more. I'm not even sure whether they had been necessary in your version or whether they have been an empty file already.

Please note: The bsp.mk is also something that is not supported by RTEMS any more. It's a compatibility hack that it's still in the `grisp2-rtems-toolchain`. Maybe it would be a good idea to remove it some-when. The usual method now is to have an application specific make file that collects the necessary flags using pkg-config. For example that one:

https://gitlab.com/c-mauderer/rtems-simulation-environment/-/blob/b4f11188ed3467a0a5808e8ab2324ffe07012825/demo/Makefile

There is also the `rtems_waf` repo that you can use if you want to write an application with a waf based build system.